liquidity/liquidity.go

```diff
@@ -556,6 +556,12 @@ func (m *Manager) dispatchBestEasyAutoloopSwap(ctx context.Context) error {
 
 	localTotal := btcutil.Amount(0)
 	for _, channel := range channels {
+		// If the channel has custom channel data, the channel is a
```

Comment:
nit: I think this code block is worth extracting to a helper function so we spare the copy-paste of the comment.